index.js

```diff
@@ -104,7 +104,11 @@ module.exports = fp(async function (fastify, opts) {
     }
     return results
   }
-}, {
+}
+
+module.exports = fp(fastifyRoutesStats, {
   fastify: '4.x',
-  name: '@fastify/route-stats'
+  name: '@fastify/routes-stats'
```

Comment:
I just want to point out, that here was a typo, which effects the exports generated by fastifyPlugin.